src/test/java/org/opensearch/knn/index/codec/KNNCodecTestUtil.java

```diff
@@ -178,6 +179,7 @@ public FieldInfo build() {
                 pointIndexDimensionCount,
                 pointNumBytes,
                 vectorDimension,
+                VectorEncoding.FLOAT32,
```

Comment:
Why we choose FLOAT32 over other value (I think it's BYTE)?

Reply:
Right now, we only support FLOAT32 - so I figured there was no sense in making it BYTE.